carma-messenger-core/j2735_convertor/src/control_message_convertor.cpp

```diff
@@ -276,13 +277,13 @@ void convert(const j2735_msgs::TrafficControlSchedule& in_msg, cav_msgs::Traffic
 // Convert cav_msgs to j2735_msgs
 ////
 
-void convert(const cav_msgs::DailySchedule& in_msg, j2735_msgs::DailySchedule& out_msg)
+void convert(const carma_v2x_msgs::msg::DailySchedule& in_msg, j2735_v2x_msgs::msg::DailySchedule& out_msg)
 {
-  out_msg.begin = in_msg.begin.toSec() / units::SEC_PER_MIN;
-  out_msg.duration = in_msg.duration.toSec() / units::SEC_PER_MIN;
+  out_msg.begin = in_msg.begin.sec / units::SEC_PER_MIN;
+  out_msg.duration = in_msg.duration.sec / units::SEC_PER_MIN;
```

Comment:
I don't think this is equivalent. The time is stored as sec and nanosec within the second. So the actual total seconds is the combination of the two. You probably want to build a time or duration object and pass in both values from the ros message then call toSec() on it.